cases/arithmetic/multiply.yaml

```diff
@@ -137,7 +137,7 @@ cases:
       description: Examples demonstrating behavior of different data types
     args:
       - value: 4.5
-        type: fp32
```

Comment:
I think here the intention was to multiple an fp32 with an fp64

Reply:
The tests seem to fail in Postgres for this option, I guessed because it did not comply with the spec, where both the operands are of fp64. What would you suggest for handling such cases.

Reply:
We could make this as an unsupported kernel pairing in postgres.yaml since postgres doesn't support fp32 right?

Reply:
ahh...my mistake...postgres does actually support fp32, it was only velox that doesn't

Reply:
made the change, thanks.